tests/functional/common/vars.sh

```diff
@@ -0,0 +1,72 @@
+# shellcheck shell=bash
+
+set -eu -o pipefail
+
+if [[ -z "${COMMON_VARS_SH_SOURCED-}" ]]; then
+
+COMMON_VARS_SH_SOURCED=1
+
+commonDir="$(readlink -f "$(dirname "${BASH_SOURCE[0]-$0}")")"
+
+# Since this is a generated filaree
```

Comment:
```suggestion
# Since this is a generated file
```

typo or filaree
![image](https://github.com/user-attachments/assets/279c4939-c3f8-4693-a6b5-5cfa366a4924)